TestBenches/MatchProcessorL3_test.cpp

```diff
@@ -0,0 +1,182 @@
+// ProjectionRouter test bench
+#include "MatchProcessorTopL3.h"
+#include "CandidateMatchMemory.h"
+#include "VMProjectionMemory.h"
+#include "ProjectionRouterBuffer.h"
+#include "VMStubMEMemoryCM.h"
+#include "FileReadUtility.h"
+#include "hls_math.h"
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <algorithm>
+#include <iterator>
+
+const int nevents = 10;  // number of events to run
```

Comment:
Why not run over all 100 events?

Reply:
Yes, it should be 100. This might be a remnant of the broken rebases from the other branch.